pkg/cloud/azure/azure_storage.go

```diff
@@ -125,13 +130,21 @@ func parseAzureURL(
 		return conf, errors.Errorf("azure uri missing %q parameter", AzureAccountNameParam)
```

Comment:
is the azure accountName even required for implicitAuth? This would be different than s3 and gcs.

Reply:
oof. that's ridiculous. could you add a note about this to where we define `AZURE_ACCOUNT_NAME`?

Reply:
sure, done.